### PR TITLE
Rename the world from DK to Workstation

### PR DESCRIPTION
**Pull request by tas50**
There is one tool and it's called Workstation. Let's pretend we never called it DK.

I'll fully admit this is "rage coding" at it's best here. We're actively pushing users to use Worksktatioin, but our docs still point them to DK. There will be no DK next year and we're setting people up for a bad time. This needs to get fixed ASAP,

Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

**Comment by kagarmoe** [comment]
I'm not opposed to this, but we need to make sure that we document the unique Workstation content in chef-web-docs.

File: chef_master/source/config_yml_kitchen.rst [hunk 1]
```diff
@@ -133,7 +133,7 @@ Kitchen can configure the chef-zero provisioner with the following Chef-specific
    * - ``chef_metadata_url``
      - **This will be deprecated in a future version.**
    * - ``chef_omnibus_install_options``
-     - Use to specify the package to be installed. Possible values: ``-P chef`` (for Chef Infra Client) and ``-P chefdk`` (for the Chef Infra Client that is packaged as part of ChefDK). Use ``-n`` to specify the nightly build. For example: ``-P chefdk`` or ``-n -P chefdk``. **This will be deprecated in a future version.** See the ``product_name``, ``product_version``, and ``channel`` settings instead.
+     - Use to specify the package to be installed. Possible values: ``-P chef`` (for Chef Infra Client) and ``-P chef-workstation`` (for the Chef Infra Client that is packaged as part of Chef Workstation). Use ``-n`` to specify the nightly build. For example: ``-P chef-workstation`` or ``-n -P chef-workstation``. **This will be deprecated in a future version.** See the ``product_name``, ``product_version``, and ``channel`` settings instead.
```
Comment (mjingle) [inline chef_master/source/config_yml_kitchen.rst:136]: Is the bolded deprecation note still applicable with this change from DK -> workstation?

File: chef_master/source/fips.rst [hunk 2]
```diff
@@ -26,7 +26,7 @@ Supported Products
 **Supported:**
 
 * `Chef Infra Client </fips.html#how-to-enable-fips-mode-for-the-chef-client>`__
-* `ChefDK </fips.html#how-to-enable-fips-mode-for-workstations>`__
+* `Chef Workstation <fips.html#how-to-enable-fips-mode-for-workstations>`__
```
Comment (mjingle) [inline chef_master/source/fips.rst]:
```suggestion
* `Chef Workstation </fips.html#how-to-enable-fips-mode-for-workstations>`__
```

File: chef_master/source/glossary.rst [hunk 3]
```diff
@@ -57,8 +57,8 @@ Glossary
 **Foodcritic**
    A linting tool for doing static code analysis on cookbooks.
 
-**kitchen**
-   Kitchen is an integration framework that is used to automatically test cookbook data across any combination of platforms and test suites. Kitchen is packaged in ChefDK.
+**Test Kitchen**
+   Test itchen is an integration framework that is used to automatically test cookbook data across any combination of platforms and test suites. Test Kitchen is packaged in Chef Workstation.
```
Comment (mjingle) [inline chef_master/source/glossary.rst]:
```suggestion
   Test Kitchen is an integration framework that is used to automatically test cookbook data across any combination of platforms and test suites. Test Kitchen is packaged in Chef Workstation.
```

File: chef_master/source/install_workstation.rst [hunk 4]
```diff
@@ -0,0 +1,143 @@
+=====================================================
+Install Chef Workstation
+=====================================================
+`[edit on GitHub] <https://github.com/chef/chef-web-docs/blob/master/chef_master/source/install_workstation.rst>`__
+
+Start your infrastructure automation quickly and easily with `Chef Workstation <https://www.chef.sh/>`__ Chef Workstation gives you everything you need to get started with Chef — ad hoc remote execution, remote scanning, configuration tasks, cookbook creation tools as well as robust dependency and testing software — all in one easy-to-install package.
```
Comment (mjingle) [inline chef_master/source/install_workstation.rst]:
```suggestion
Start your infrastructure automation quickly and easily with `Chef Workstation <https://www.chef.sh/>`__ . Chef Workstation gives you everything you need to get started with Chef — ad hoc remote execution, remote scanning, configuration tasks, cookbook creation tools as well as robust dependency and testing software — all in one easy-to-install package.
```

File: chef_master/source/config_yml_kitchen.rst (hunk 1 above)
Comment (tas50) [reply chef_master/source/config_yml_kitchen.rst:136]: Should still be valid

**Review by mjingle — APPROVED** [review]
tenor-192699211

**Pull request merged**